packages/@okta/vuepress-site/code/javascript/okta_sign-in_widget/index.md

```diff
@@ -308,6 +307,61 @@ function error(err) {
 }
 ```
 
+
+## Using with Okta's SDKs
+
+Okta provides a number of SDKs that you might want to use the Sign-In Widget with, including Angular, React, and Vue.
+
+Using the Sign-In Widget with our SDKs that target the web is fairly straightforward.
+
+### Angular
+
+The [Okta Sign-In Widget and Angular guide](/code/angular/okta_angular_sign-in_widget/) shows the code you'll need to embed the Sign-In Widget in an Angular app.
```

Comment:
Just to avoid possible ambiguity: "the code you'll need to embed the Sign-In Widget" > "the code you'll need **in order** to embed the Sign-In Widget" (and same for this phrase in the sections for the other SDKs).